tests/online_migration/sle12/yast2_migration.pm

```diff
@@ -0,0 +1,77 @@
+use base "installbasetest";
+use strict;
+use testapi;
+
+sub run {
+    my $self = shift;
+    script_run "yast2 migration";
+
+    # ask installing update before migration if not perform full update for system
+    if (!get_var("FULL_UPDATE_BEFORE_MIGRATION")) {
+        assert_screen 'yast2-migration-onlineupdates', 10;
+        send_key "alt-y";
+        assert_screen 'yast2-migration-updatesoverview', 10;
+        send_key "alt-a";
+    }
+
+    # wait for migration target after needed updates installed
+    assert_screen 'yast2-migration-target', 300;
+    send_key "alt-n";
+    assert_screen 'yast2-migration-installupdate', 200;
+    send_key "alt-y";
+    assert_screen 'yast2-migration-proposal', 200;
+
+    # disable installation repo
+    if (check_screen("recommend-to-disable", 5)) {
```

Comment:
Does this is only happen sometimes? what are the criteria for YaST migration to ask for disabling repos? or should it always ask?